Gemfile.lock

```diff
@@ -394,4 +394,4 @@ RUBY VERSION
    ruby 2.7.2p137
 
 BUNDLED WITH
-   2.2.6
+   2.2.6
```

Comment:
Can you check if there is a trailing space here, and if there is, remove it and push the change to github ? It makes the PR simpler